src/reference/asciidoc/si-kafka.adoc

```diff
@@ -277,6 +277,22 @@ In most cases, this will be an `ErrorMessageSendingRecoverer` which will send th
 When building `ErrorMessage` (for use in the `error-channel` or `recovery-callback`), you can customize the error message using the `error-message-strategy` property.
 By default, a `RawRecordHeaderErrorMessageStrategy` is used; providing access to the converted message as well as the raw `ConsumerRecord`.
 
+Starting with _Spring for Apache Kafka version 2.2_ (_Spring Integration Kafka 3.1_), the container factory used for `@KafkaListener` annotations can also be used to create `ConcurrentMessageListenerContainer` s for other purposes.
+See <<container-factory>> for an example.
+
+With the Java DSL, the container does not have to be configured as a `@Bean` because the DSL will register the container as a bean.
```

Comment:
I think the same concern here as well.

Reply:
? Scroll up a few lines...

>When using Spring Integration Java DSL:

I think we can just remove the reference to the DSL in the pure Kafka section.